virttest/guest_agent.py

```diff
@@ -885,6 +885,42 @@ def get_disks(self):
         self.check_has_command(cmd)
         return self.cmd(cmd)
 
+    def ssh_add_authorized_keys(self, username, *keys, **kwargs):
+        """
+        Add ssh public keys into guest-agent.
+
+        :param username: username of host
+        :param keys: value of public keys
+        :param reset: bool value to enable reset keys, defaults to false
```

Comment:
Please update docstring according to new arguments.

Reply:
okay, got it.